dockstore-webservice/src/main/java/io/dockstore/webservice/core/Entry.java

```diff
@@ -306,6 +306,28 @@ public abstract class Entry<S extends Entry, T extends Version> implements Compa
     @Transient
     private Set<OrcidAuthor> orcidAuthors = new HashSet<>();
 
+    @JsonIgnore
+    @Column(nullable = true, columnDefinition = "varchar(32)")
+    @Schema(description = "The visibility of the Git repo")
+    @Enumerated(EnumType.STRING)
```

Comment:
(no action needed) was reading the other day about Postgres's native enum type, https://www.postgresql.org/docs/9.1/datatype-enum.html
It seems a pain to use through JPA/Hibernate https://thorben-janssen.com/hibernate-enum-mappings/ but something to keep an eye on